bin/memex/hackathon_2016_07/notebooks/CP1 Data Curation.ipynb

```diff
@@ -16,7 +16,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 34,
```

Comment:
As possible sanitation for the future, I try to clear all cell outputs before saving them for check in for git.  This should prevent superfluous updates to run counts and output results.